cpp/oneapi/dal/detail/cpu_info_impl.cpp

```diff
@@ -80,6 +81,9 @@ void cpu_info_impl::print_any(const std::any& value, std::stringstream& ss) cons
     else if (ti == typeid(cpu_vendor)) {
         print<cpu_vendor>(value, ss);
     }
+    else {
+        throw unimplemented{ dal::detail::error_messages::unsupported_data_type() };
```

Comment:
Can you pass the typeid of `ti` so that this message is a little more specific? It would be nice to know what the unknown type is